docs/protocol/src/dex/action/swap_claim.md

```diff
@@ -64,6 +65,20 @@ using the witnessed values above and where `ds` is a constant domain separator:
 
 as described in [Nullifiers](../notes/nullifiers.md).
 
+### Nullifier Key Linking
+
+The zk-SNARK certifies that the diversified address $pk_d$ associated with the note being spent was derived as:
```

Comment:
This doesn't seem right -- a `SwapClaim` doesn't spend any notes, so there is no address associated with the note being spent.

What we need to describe here is:

- Which `nk` is permitted to derive the nullifier for the state commitment consumed by the `SwapClaim`?
- How does the proof statement certify that the revealed nullifier was correctly derived using the permitted `nk`?

Reply:
Yeah, this was a mistake, there's an address associated with the swap being claimed.